azurerm/resource_arm_virtual_machine_scale_set_migration.go

```diff
@@ -2,16 +2,18 @@ package azurerm
 
 import (
 	"fmt"
-	"log"
-
 	"github.com/hashicorp/terraform/terraform"
+	"log"
 )
 
 func resourceVirtualMachineScaleSetMigrateState(v int, is *terraform.InstanceState, meta interface{}) (*terraform.InstanceState, error) {
 	switch v {
 	case 0:
 		log.Println("[INFO] Found AzureRM Scale Set State v0; migrating to v1")
 		return resourceVirtualMachineScaleSetStateV0toV1(is, meta)
+	case 1:
+		log.Printf("[INFO] Found AzureRM Scale Set State v0; migrating to v2")
```

Comment:
I would keep the `log.Println` as the one above. Unless you expect to pass some format string, and in which case it would be fine.

Reply:
totally miss that.